src/com/JBC2/Task24/TimeInterval.java

```diff
@@ -0,0 +1,65 @@
+package com.JBC2.Task24;
+
+import java.sql.Time;
+
+//Создать класс и объекты описывающие промежуток времени. Сам промежуток
+//в классе должен задаваться тремя свойствами: секундами, минутами, часами.
+//Сделать методы для получения полного количества секунд в объекте, сравнения
+//двух объектов (метод должен работать аналогично compareTo в строках). Создать
+//два конструктора: получающий общее количество секунд, и часы, минуты и секунды
+//по отдельности. Сделать метод для вывода данных.
+public class TimeInterval {
+    int hour;
+    int min;
+    int sec;
+
+    TimeInterval(int s) {      //конструтор получающий количество секунд
+        sec = s % 60;
```

Comment:
better in constructor use this for variables
this.sec = s % 60;
this.min = m % 60;
...